modules/scan-inactive-images.adoc

```diff
@@ -2,15 +2,27 @@
 //
 // * operating/manage-vulnerabilities.adoc
 // * operating/examine-images-for-vulnerabilities.adoc
-:_module-type: PROCEDURE
+:_content-type: PROCEDURE
 [id="scan-inactive-images_{context}"]
 = Scanning inactive images
 
-{product-title} scans all active (deployed) images every 4 hours and updates the image scan results to reflect the latest vulnerability definitions.
+{rh-rhacs-first} scans all active (deployed) images every 4 hours and updates the image scan results to reflect the latest vulnerability definitions.
 //TODO: add link to Re-scanning images.
 
-You can also configure {product-title} to scan inactive (not deployed) images automatically.
+You can also configure {product-title-short} to scan inactive (not deployed) images automatically.
 
+
+.Procedure
+
+. On the RHACS portal, navigate to *Vulnerability Management > Dashboard*.
+. On the *Dashboard* view header, select *IMAGES*.
+. Click *MANAGE WATCHES* to manage the scanning of watched images.
+. In the *MANAGE WATCHED IMAGES* dialog, enter the inactive image’s name, and not the image `id`, for which you want to enable scanning. Image name is the fully-qualified image name, beginning with the registry and ending with the tag.
+. Select *ADD IMAGE*. RHACS then scans the image and shows the error or success message.
+. Select *RETURN TO IMAGE LIST* to view the *IMAGES* page.
+
```

Comment:
I recommend you add a note for the "remove watch".  The image removed from the watch list may not be removed from the image list page immediately. The images are removed once the retention configuration threshold (Platform Configuration -> System Configuration) is met.

Reply:
Yes, please add a note.